src/components/__tests__/MentorBanner.test.tsx

```diff
@@ -0,0 +1,75 @@
+import { render, screen } from '@testing-library/react';
+import mediaQuery from 'css-mediaquery';
+import React from 'react';
+
+import { MentorBanner } from '../MentorBanner';
+
+export const createMatchMedia =
+  (width: number) =>
+  (query: string): MediaQueryList => ({
+    matches: mediaQuery.match(query, { width }),
+    media: query,
+    onchange: null,
+    addListener: () => jest.fn(),
+    removeListener: () => jest.fn(),
+    addEventListener: jest.fn(),
+    removeEventListener: jest.fn(),
+    dispatchEvent: jest.fn(),
+  });
+
+const BannerProps = {
+  title: 'Banner Title',
+  description: 'This is a description',
+  images: [
+    {
+      path: '/desktop.jpg',
+      alt: 'alt desktop text',
+      type: 'desktop',
+    },
+    {
+      path: '/mobile.jpg',
+      alt: 'alt mobile text',
+      type: 'mobile',
```

Comment:
We also have tablet.

Reply:
@dricazenck Does the tablet have a different image for the banner? There were two images in the Figma, one for desktop and another one for mobile.

Reply:
Right now we don't have it, but that could change in the future.